app/routes/planet_routes.py

```diff
@@ -1,30 +1,89 @@
-from flask import Blueprint, abort, make_response
-from app.models.planet import planets
+from flask import Blueprint, abort, make_response, request, Response
+from app.models.planet import Planet
+from ..db import db
 
 # created blueprint
-planets_bp = Blueprint("planets_bp",__name__, url_prefix=("/planets"))
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+@planets_bp.post("")
+def create_planet():
+    request_body = request.get_json()
+    name = request_body["name"]
+    description= request_body["description"]
+    size_dia_km = request_body["size_dia_km"]
+
+    new_planet = Planet(name=name, description=description, size_dia_km=size_dia_km)
+    db.session.add(new_planet)
+    db.session.commit()
+
+    response = new_planet.to_dict()
+    return response, 201
 
 @planets_bp.get("")
 def get_all_planets():
-    results_list = []
 
-    for planet in planets: 
-        results_list.append(planet.to_dict())
-    return results_list
+    query = db.select(Planet)
+
+    name_param = request.args.get("name")
```

Comment:
👍 Nice filter parameter behavior.

For numeric values, a `like` comparison is probably not what most clients would expect. It can often be useful to allow filtering for equal to, less than, or greater than a value. How might we allow a client to specify such behavior?